File: beecrowd/1005/1005 - Average 1 .cs
using System;

class Program
{
	static void Main(string[] args)
	{
		double A, B, average;
		
		A = double.Parse(Console.ReadLine());
		B = double.Parse(Console.ReadLine());
		
		average = ((A * 3.5) + (B * 7.5)) / 11;
		
		Console.WriteLine($"MEDIA = {average:F5}");
	}
}

File: beecrowd/1005/1005 - Average 1 .c
#include <stdio.h>

int main()
{
	float A, B, average;
	
	scanf("%f", &A);
	scanf("%f", &B);
	
	average = ((A * 3.5) + (B * 7.5)) / 11;
	
	printf("MEDIA = %.5f\n", average);

	return 0;
}